{"name": "User with wrong data types fails validation", "status": "passed", "description": "Test that wrong data types fail validation.", "attachments": [{"name": "Schema Validation Failure", "source": "8df97c90-450e-4b47-b3d0-be70e7c1fdb0-attachment.json", "type": "application/json"}, {"name": "Schema Validation Failure", "source": "b3b1f824-30d5-4684-b59a-de57bd4c0870-attachment.json", "type": "application/json"}], "start": 1756578178225, "stop": 1756578178230, "uuid": "67ae55f6-41ab-424d-bbbf-a7844b13654d", "historyId": "71818746d71d76d5be64a23650196b19", "testCaseId": "71818746d71d76d5be64a23650196b19", "fullName": "tests.test_schema_validation.TestSchemaValidation#test_wrong_data_types_fail_validation", "labels": [{"name": "story", "value": "User Schema"}, {"name": "feature", "value": "Schema Validation"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_schema_validation"}, {"name": "subSuite", "value": "TestSchemaValidation"}, {"name": "host", "value": "Dons-MacBook-Air.local"}, {"name": "thread", "value": "83619-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_schema_validation"}], "titlePath": ["tests", "test_schema_validation.py", "TestSchemaValidation"]}